frontend/src/NavigationMenu/NavigationMenu.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import { List, ListItem, ListItemButton, ListItemText, Box, Drawer } from '@mui/material'
+import { UserType, UserTypeProps } from '../types/global.d'
+import { useTranslation } from 'react-i18next'
+
+export default function NavigationMenu(props: UserTypeProps) {
+    const drawerWidth: number = 200
+    const { t } = useTranslation('translation', { keyPrefix: "translation.page_names" })
+    // now you can test menu translation by going on /login page and press "Submit" button
+    let menu_button = (text: string, path: string) => (
+        <ListItem disablePadding>
+            <ListItemButton href={path}>
+                <ListItemText primary={t(text)} />
+            </ListItemButton>
+        </ListItem>
+    )
+    let projects, leaders, statistics, settings, users, tags, handbook
```

Comment:
Why do you split definition and declaration? Mb just like:
```
let projects = menu_button('projects', "/projects")
let leaders = ...
```
And where are types? It all should be ReactComponent or smth like this...